features/meta.yml

```diff
@@ -19,4 +20,3 @@ compat_features:
   - html.elements.meta.name
   - html.elements.meta.name.color-scheme
```

Comment:
Can you exclude color-scheme? It was introduced around the time of the CSS property by the same name and should probably be bunched in with that feature, or be a new feature.

Reply:
done.